py3status/modules/backlight.py

```diff
@@ -3,19 +3,34 @@
 Display the current screen backlight level.
 
 Configuration parameters:
+    brightness_delta: Change the brightness by this step
+        (default 5)
+    brightness_initial: Set brightness to this value on start
+        (default 100)
```

Comment:
how about having this default as None, but if set then will set that value at startup?  That way we have one less parameter.